modules/node-observability-scripting.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-observability-scripting_{context}"]
+= Node Observability Operator Scripting
```

Comment:
The main section and this procedure headings are same. 
This needs to begin with a gerund since this is a procedure.

Reply:
Understood.